src/components/TextInput/TextInput.stories.tsx

```diff
@@ -1,21 +1,21 @@
 import type { Meta, StoryObj } from '@storybook/react';
-import { TextInput } from '~/src/index';
+import { Button, TextInput } from '~/src/index';
 
 const meta: Meta<typeof TextInput> = {
   title: 'Components (Draft)/Text inputs',
   component: TextInput,
   argTypes: {
-    width: { control: 'select' },
+    fullWidth: { control: 'boolean' },
     isDisabled: { control: 'boolean' }
   },
   parameters: {
     docs: {
       description: {
         component: `
-### CFPB DS - TextInput component
+  Text inputs allow the user to enter any combination of letters, numbers, or symbols. Text input fields can span single or multiple lines.
```

Comment:
I prefer having these implemented as JSDOC comments for the actual component, similar to the this [Banner](https://github.com/cfpb/design-system-react/blob/main/src/components/Banner/Banner.tsx#L33C4-L37) example (you'd need to restart the local app for Storybook to pick up the change).  That allows these bits of documentation to be visible to code maintainers *and* to serve as the intro paragraphs within Storybook, while only having to maintain it in one place.